hudi-cli/src/main/java/org/apache/hudi/cli/commands/CompactionCommand.java

```diff
@@ -220,13 +220,13 @@ public String compact(
       @ShellOption(value = {"--parallelism"}, defaultValue = "3",
           help = "Parallelism for hoodie compaction") final String parallelism,
       @ShellOption(value = "--schemaFilePath",
-          help = "Path for Avro schema file", defaultValue = ShellOption.NULL) final String schemaFilePath,
+          help = "Path for Avro schema file", defaultValue = "") final String schemaFilePath,
```

Comment:
"--schemaFilePath" is not required, if it is not in args , we will get an exception "'arg' must not be null.".